test/testenv/mcutil.go

```diff
@@ -163,3 +166,21 @@ func GetPodIP(ns string, podName string) string {
 	}
 	return restResponse.Status.PodIP
 }
+
+// GetMCConfigMap gets config map for give Monitoring Console Name
+func GetMCConfigMap(deployment *Deployment, ns string, mcName string) (*corev1.ConfigMap, error) {
+	mcConfigMapName := enterprise.GetSplunkMonitoringconsoleConfigMapName(deployment.GetName(), enterprise.SplunkMonitoringConsole)
+	mcConfigMap, err := GetConfigMap(deployment, ns, mcConfigMapName)
+	if err != nil {
+		logf.Log.Error(err, "Failed to get MC Conig Map")
+		return mcConfigMap, err
+	}
+	logf.Log.Info("MC Config Map contents", "Data", mcConfigMap.Data)
+	return mcConfigMap, err
+}
+
+// CheckPodNameInString checks for pod name in string
+func CheckPodNameInString(podName string, configString string) bool {
+	logf.Log.Info("Check MC Config String has Pod configured", "MC CM String", configString, "POD String", podName)
```

Comment:
Can we elaborate on this log? "MC CM String" is unclear

Reply:
Will do. This however common terminology we use internally however I understand how this can be confusing.